InnerEye/ML/run_ml.py

```diff
@@ -242,14 +242,10 @@ def is_offline_run(self) -> bool:
         return is_offline_run_context(RUN_CONTEXT)
 
     @property
-    def innereye_config(self) -> DeepLearningConfig:
+    def innereye_config(self) -> Optional[DeepLearningConfig]:
         """
-        Gets the model configuration object for all built-in InnerEye models. Raises an exception if the present
-        object trains a LightningContainer that is not a built-in InnerEye model.
+        Gets the model configuration object for all built-in InnerEye models.
```

Comment:
is that change really necessary anymore?

Reply:
No I don't think we do need that, you are correct. Fixed